src/shared/components/RadioButton/RadioButton.tsx

```diff
@@ -1,35 +1,29 @@
 import React from 'react'
-import { Label, Input, StyledInput, RadioButtonStyleProps } from './RadioButton.style'
+import { Label, Input, StyledInput, StyledLabelText, RadioButtonStyleProps } from './RadioButton.style'
 
 type RadioButtonProps = Partial<{
+  selected: string | number
```

Comment:
Maybe we could call this something like `selectedValue`? This may suggest it's a boolean, like `isSelected`